src/handler/messagehandler.ts

```diff
@@ -58,7 +58,7 @@ export default class MessageHandler {
         img = await new Promise((resolve) => {
           // @ts-ignore
           request.get(attachment.url, async (err: any, res: any, body: Buffer) => {
-            if (body !== undefined) {
+            if (body != null) {
```

Comment:
Why `!=` and not `!==`?

Reply:
I believe in runtime I was finding it to be null, while the code was checking for undefined. I went for the best of both worlds and went with an assertation that accounts for both null as well as undefined.